manifests/04-operator.yaml

```diff
@@ -32,9 +32,6 @@ spec:
       containers:
         - name: cluster-node-tuning-operator
           image: registry.svc.ci.openshift.org/openshift/origin-v4.0:cluster-node-tuning-operator
-          ports:
```

Comment:
Hey @jmencak, why did you get rid of the metrics port?

Reply:
It shouldn't have been there in the first place as it is currently unused.  must-gather tool chokes on this, tries to gather the metrics and exits 1.  Hence the removal.